pkg/restapi/operation/openapi.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package operation
+
+import "github.com/trustbloc/edv/pkg/restapi/models"
+
+// genericError model
+//
+// swagger:response genericError
+type genericError struct { // nolint: unused,deadcode
+	// in: body
+	ErrMsg string
+}
+
+// createVaultReq model
+//
+// swagger:parameters createVaultReq
+type createVaultReq struct { // nolint: unused,deadcode
+	// in: body
+	NewVaultRequest models.DataVaultConfiguration
+}
+
+// createVaultRes model
+//
+// swagger:response createVaultRes
+type createVaultRes struct { // nolint: unused,deadcode
+	// For some reason this doesn't display in the Swagger response UI currently...
+	// To see the returned location, you'll have to open the developer console.
+	// The location will be in the form of HostURL/encrypted-data-vaults/vaultID
+	Location string
+}
+
+// queryVaultReq model
+//
+// swagger:parameters queryVaultReq
+type queryVaultReq struct { // nolint: unused,deadcode
+	// in: path
+	// required: true
+	VaultID string `json:"vaultID"`
+	// in: body
+	QueryRequest models.Query
+}
+
+// queryVaultRes model
+//
+// swagger:response queryVaultRes
+type queryVaultRes struct { // nolint: unused,deadcode
+	// in: body
+	QueryResults string
```

Comment:
Shouldn't this be a JSON rather than string ? Is this a part of https://github.com/trustbloc/edv/issues/90 ?

Reply:
Yeah, I was planning to do that as a part of #90. I'll update that story to explicitly mention that.

Reply:
add a TODO here

Reply:
Done. The response is actually just an array per the spec, so I'll have to see if I can define it as an array in the OpenAPI annotation.